about.html

```diff
@@ -0,0 +1,138 @@
+                <div class="navbar-nav">
+                    <a class="nav-link active" aria-current="page" href="index.html">Home</a>
+                    <a class="nav-link" href="about.html">About Programme</a>
+                  <a class="nav-link" href="#">Designs</a>
+                  <a class="nav-link createborder" href="#">REGISTER NOW</a>
+                </div>
+              </div>
+            </div>
+          </nav>
+        </header>
+    </section>
+
+    <section class="aboutbg">
+        <div class="container">
+            <h1>GLOBBERY FASHION <br/> TRAINING PROGRAMME 2022</h1>
+            <p class="subtitle_abt">Our training programme will kick off with lots of exerpt and practical session,
+                you will learn the basis of cuting and sketching, Join the moving train and move your business to the 
+                next level. You can stop learning. If you do then fashion will move past you. 
+                We are also having different session with different sessoned professionals and designers all over the country.
+                Don't miss out on this opportunity to take your career to the next level.
+            </p>
+            <div class="bg-white-color">Please contact us more questions in Globbery fashion training 2022</div>
+            <div class="bg-white-color">services@globerry.org</div>
+        </div>
+    </section>
+
+
+    <section class="abt-session">
+        <div class="container">
+            <div class="row">
+                
+                <h6> Globbery Training and Fashion School 2022</h6>
+                <div class="col-12 exerpt-content  mx-auto">
+                   <p>This training is a once in a life time. Is a training that will make you a better person.
+                       Don't miss out on this opportunity of making it. Our programme will start with intense
+                       training session and continue to ensure that everything is going fine.
+                   </p>
+                 </div>
+
+                 <div class="col-12">
+                     <img src="./images/logo6.png" class="making-border img-fluid" alt="Globerry Logo"/>
+                 </div>
+                 
+            </div>     
+        </div>
+    </section>
+
+
+    <section class="past-trainings">
+        <h3 class="text-center">See Past Globery Trainings</h3>
+        <p>Take a look at the two(2) Globerry Couture which took place at Lameridian Hotel, Igoja Lagos.</p>
+        
+        <div class="fashion-events">
+            <div class="wrapper">
+                <div class="content">
+                 <p>2012</p>
+                 <div>Designers Trend for Fashion Longe</div>
+                </div>
+            </div>
+
+            <div class="wrapper2">
+                <div class="content2">
+                 <p>2013</p>
+                 <div>Training Session with Indian Tailors</div>
+                </div>
+            </div>
```

Comment:
- [ ] According to the guidelines, this section should only contain 2 images with text centered over them. At the moment you have 3. Please remove one.
- [ ] On mobile devices, the articles should be organized in a vertical column. Right now they are in a row and it doesn't look very good 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/169616519-d7cea48f-5053-4db7-83dd-d7e262f05c61.png)
### Guideline
![image](https://user-images.githubusercontent.com/46839548/169616713-e0404028-edfb-4127-8bab-e7e884ab17c2.png)
- [ ] [Optional] Using semantic HTML, you could replace each `<div>` tag inside this section with an `<article>` tag.